website/templates/index.html

```diff
@@ -35,12 +35,10 @@ <h4 class="card-title">{% trans "Number Error" %}</h4>
                       </div>
                     </div>
                     <ul class="list-group list-group-flush">
-                        <li class="list-group-item">
-                            <a href="/search/?query=1&type=label">Examples</a>
-                        </li>
+                        <a class="btn btn-default" href="/search/?query=1&type=label">Examples</a>
                     </ul>   
                 </div>
-                <div class="col-sm-4">    
+                <div class="col-sm-3 cborder">
```

Comment:
I have changed from 4 to 3 so there is some space between boxes without too much padding needed or risk of them overflowing. I have added margins to ensure they don't shift.